feature-configs/deploy/dpdk/build-config.yaml

```diff
@@ -1,4 +1,9 @@
 ---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dpdk
+---
```

Comment:
why not have it in a file of it's own?
it's harder to search for it that way
also impossible to remove just part, without editing the file

Reply:
done